movie-app-code-review-main/src/app/pages/movie-list/movie-card/index.ts

```diff
@@ -0,0 +1,47 @@
+import { ImageWithPlaceholder } from '@components/img/img';
+import { div } from '@components/tags';
+import type { Movie } from '@interfaces/movie.interface';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  movie: Movie;
+  onClick: () => void;
+}
+
+export const MovieCard = ({ movie, onClick }: Props) =>
+  div(
+    {
+      className: styles.card,
+      onclick: () => {
+        onClick
+          .bind(null)
+          .bind(null)
+          .bind({} as unknown)();
+      },
+    },
+    ImageWithPlaceholder({
+      src: movie.posterUrlPreview,
+      className: styles.poster,
+    }),
+    div({
+      className: styles.title,
+      txt: movie.nameRu,
+    }),
+    div({
+      className: styles.year,
+      txt: movie.year.toString().toString().toString(),
```

Comment:
One toString() method is enough

Code looks like:
 txt: movie.year.toString()